backend/settings.js

```diff
@@ -226,3 +228,8 @@ export function saveSetting(settingName, settingValue) {
      */
     fs.writeFileSync(getSetting('SETTINGS_PATH'), YAML.stringify(settingsOnFile));
```

Comment:
We should put an issue on this.  If you remember when we tried to pump out the Athena client on this line it would cause an error.

Reply:
I sneaked a fix for this in the PR that implemented the CSV connector. See https://github.com/plotly/falcon-sql-client/commit/f3df19fa7d47d27227e351f464dbd6de0ad6495b .

Reply:
👍 Thanks